README.md

```diff
@@ -539,67 +531,67 @@ stopAnim(
 	ped,		-- Optional, for if you want any one other than the player to do the animation
 )
 ```
-### `makeVeh(model, coords)`
-Spawns a vehicle for the player to use
-- Server Synced
-- Easy creation
-- Returns entity id for further control through the script
-- Loads model before spawning
-- Unloads model from memory cache after spawn
-
-Example of use:
-```lua
-local vehicle = makeVeh(
-	`zentorno`,
-	vec4(-596.74, 2090.99, 131.41, 16.6)
-)
-print(vehicle, GetEntityCoords(vehicle))
-```
-
-### `makePed(model, coords, freeze, collision, scenario, anim, synced)`
-Spawns a controllable ped
-- Loads the model before spawning
-- Unloads model from memory cache after spawn
-- Several options for creation
-- Can spawn with scenario name or anims
-- Spawns invincible
-
-Example of use:
-```lua
-local ped = makePed(
-	`MP_M_Freemode_011,
-	vec4(-596.74, 2090.99, 131.41, 16.6),
-	true,
-	false,
-	nil,	
-	{ "amb@prop_human_parking_meter@male@idle_a", "idle_a" },
-	false
-)
-print(ped, GetEntityCoords(ped))
-```
+### `makeVeh(model, coords)`
+Spawns a vehicle for the player to use
+- Server Synced
+- Easy creation
+- Returns entity id for further control through the script
+- Loads model before spawning
+- Unloads model from memory cache after spawn
+
+Example of use:
+```lua
+local vehicle = makeVeh(
+	`zentorno`,
+	vec4(-596.74, 2090.99, 131.41, 16.6)
+)
+print(vehicle, GetEntityCoords(vehicle))
+```
+
+### `makePed(model, coords, freeze, collision, scenario, anim, synced)`
+Spawns a controllable ped
+- Loads the model before spawning
+- Unloads model from memory cache after spawn
+- Several options for creation
+- Can spawn with scenario name or anims
+- Spawns invincible
+
+Example of use:
+```lua
+local ped = makePed(
+	`MP_M_Freemode_011,

```

Comment:
Not your fault, I respect it was already there but let's fix this documentation once and for all

```suggestion
	`MP_M_Freemode_011`,

```